lib/core/db_models/worker/schema/newsNotificationSchema.dart

```diff
@@ -0,0 +1,15 @@
+class NewsNotificationSchema {
```

Comment:
guessing this is a mistake and we shouldn't have an underscore between these words

Reply:
sam these are the keys  return from the api and it should be same as mentioned in the api to parse the data